test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -128,18 +129,52 @@ var (
 	parentResources = map[schema.GroupVersionResource]schema.GroupVersionResource{
 		gvr("extensions", "v1beta1", "replicationcontrollers/scale"): gvr("", "v1", "replicationcontrollers"),
 	}
+
+	nonPersistentObjects = map[schema.GroupVersionResource]etcd.StorageData{
```

Comment:
nit: `nonPersistentReviewObjects` (to distinguish between this and other non-persistent subresources)

Reply:
simplify this to `map[schema.GroupVersionResource]string`, no need to nest the Stub field

Reply:
I would prefer to keep the same name `nonPersistentObjects` in case GetStubObject can be used as a generic func to get stubs either from persistent or a non persistent data sources. If Reviews specific name is used then adding other non persistent type of object would require change of func..

Reply:
ok